docs/self-managed/setup/upgrade.md

```diff
@@ -2,130 +2,61 @@
 id: upgrade
 title: "Camunda 8 Helm upgrade"
 sidebar_label: "Upgrade"
-description: "To upgrade to a more recent version of the Camunda Helm charts, there are certain things you need to keep in mind."
+description: "Upgrade to a more recent version of the Camunda Helm charts."
 ---
 
-To upgrade to a more recent version of the Camunda Helm charts, configuration adjustments may be required. Review the [instructions for specific versions](#version-update-instructions) prior to starting an upgrade.
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
 
 :::note
-The recommended Helm upgrade path is to the **latest patch** release of the **next major version**.
+When upgrading to a new version of the Camunda 8 Helm charts, we recommend updating to the **latest patch** release of the **next major version**.
 :::
 
-### Upgrading where Identity disabled
+Upgrading between minor versions of the Camunda helm chart may require [configuration changes](#update-your-configuration). To upgrade between patch versions or when no configuration changes are required, see the [`helm upgrade`](#identity-disabled) instructions.
```

Comment:
```suggestion
Upgrading between minor versions of the Camunda Helm chart may require [configuration changes](#update-your-configuration). To upgrade between patch versions or when no configuration changes are required, see the [`helm upgrade`](#identity-disabled) instructions.
```

Cap Helm Chart

Reply:
Actually, as I'm reading further, I might be misunderstanding when we capitalize Helm and when we don't. 🤔 This would be a good candidate for getting into our [cheatsheet](https://github.com/camunda/camunda-docs/blob/main/howtos/technical-writing-cheatsheet.md#product-names-and-other-terminology).

Reply:
The [Helm docs](https://helm.sh/docs/chart_best_practices/conventions/#usage-of-the-words-helm-and-chart) say **Helm** gets capitalized, **chart** does not - I'll update this!

Definitely a good idea for the cheatsheet, as we definitely do not standardize throughout the docs just yet. 😅